lib/github.rb

```diff
@@ -55,7 +55,9 @@ def self._pr_data(pr)
     end
 
     result["reviewRequests"] = pr["reviewRequests"]["nodes"].map do |rr|
-      { "user" => name_and_login(rr["requestedReviewer"]) }
+      if !rr["requestedReviewer"].nil?
```

Comment:
Do you have an example PR where this broke? There should be no nodes returned if the `requestedReviewer` is nil

Reply:
The failing PR is this one: https://github.com/Appboy/currents-users-events/pull/216

Reply:
It seems that one of my team members has contributions outside of Braze being picked up

Reply:
It seems that one of my team members has contributions outside of my organization being picked up

Reply:
I see, there is a review request, but I don't have access to that team and apparently the API is returning null even though the team name is visible in the web UI